README.md

```diff
@@ -31,7 +31,6 @@ jobs:
       - name: Deploy
         uses: cloudflare/wrangler-action@v3
         with:
-          packageManager: pnpm # you can omit this if you use npm
```

Comment:
+1, we should be forcing this on devs

Reply:
typo, and I assume you mean "shouldn't"? If so, agreed!

Reply:
oops.. yup. shouldn't